partner_multi_relation/README.rst

```diff
@@ -0,0 +1,104 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=================
+Partner Relations
+=================
+
+This module aims to provide generic means to model relations between partners.
+
+Examples would be 'is sibling of' or 'is friend of', but also 'has contract X
+with' or 'is assistant of'. This way, you can encode your knowledge about your
+partners directly in your partner list.
+
+Installation
+============
+
+Configuration
+=============
+
+Usage
+=====
+
+Before being able to use relations, you'll have define some first. Do that in
+Sales / Configuration / Address Book / Partner relations. Here, you need to
+name both sides of the relation: To have an assistant-relation, you would name
+one side 'is assistant of' and the other side 'has assistant'. This relation
+only makes sense between people, so you would choose 'Person' for both partner
+types. For the relation 'is a competitor of', both sides would be companies,
+while the relation 'has worked for' should have persons on the left side and
+companies on the right side. If you leave this field empty, the relation is
+applicable to all types of partners.
+
+If you use categories to further specify the type of partners, you could for
```

Comment:
If you use tags instead of categories?
Maybe you can also edit the label in the form